src/pages/AnimeKO/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from '../pageInterface';
+
+let jsonData;
+
+export const AnimeKO: pageInterface = {
+  name: 'AnimeKO',
+  domain: 'https://animeko.co',
+  languages: ['French'],
+  type: 'anime',
+  isOverviewPage() {
+    return !!j.$('.showcase-shadow').length;
+  },
+  isSyncPage() {
+    return !!j.$('.player-wrap').length;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.small-card h2').text().trim();
+    },
+    getIdentifier(url) {
+      return utils
+        .absoluteLink(j.$('.small-card h2 a').attr('href'), AnimeKO.domain)
+        .split('/')
+        .pop();
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(j.$('.small-card h2 a').attr('href'), AnimeKO.domain);
+    },
+    getEpisode(url) {
+      return Number(j.$('.player-releases :selected').text().split(' ')[1]);
+    },
+    nextEpUrl(url) {
+      return utils.absoluteLink(j.$('.btn-next').attr('href'), AnimeKO.domain);
+    },
+    getMalUrl(provider) {
+      return 'https://myanimelist.net/anime/' + jsonData.mal_id;
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j.$('.showcase h1').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getMalUrl() {
+      return 'https://myanimelist.net/anime/' + jsonData.mal_id;
+    },
+    uiSelector(selector) {
+      j.$('.showcase');
+    },
+    list: {
+      offsetHandler: true,
+      elementsSelector() {
+        return j.$('.small-card');
+      },
+      elementEp(selector) {
+        return Number(selector.find('span.badge-number').text());
+      },
+      elementUrl(selector) {
+        return utils.absoluteLink(selector.find('a').attr('href'), AnimeKO.domain);
+      },
+    },
+  },
+  init(page) {
+    j.$(function () {
+      api.storage.addStyle(
+        require('!to-string-loader!css-loader!less-loader!./style.less').toString(),
+      );
+      j.$(function () {
+        if (j.$('#malsync').length) {
+          jsonData = JSON.parse(j.$('#malsync').text());
+        }
+      });
+      page.handlePage();
+      utils.urlChangeDetect(function () {
```

Comment:
As far as I see does the page always reload/new page, so listening for url changes is not necessary?